src/main/java/edu/gatech/cs2340/trydent/sample/IndexWrapExample.java

```diff
@@ -0,0 +1,184 @@
+package edu.gatech.cs2340.trydent.sample;
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.RenderingHints;
+import java.awt.image.BufferedImage;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+import javax.swing.JComponent;
+import javax.swing.JFrame;
+
+import com.sun.javafx.geom.Rectangle;
```

Comment:
Is there a reason why you are using the JavaFX private API instead of the public one?